<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MobiiContactInformations</class>
 <widget class="QWidget" name="MobiiContactInformations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>482</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MobiiContactInformations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="contactDetailsContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="pictureLabel">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>140</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>140</height>
           </size>
          </property>
          <property name="pixmap">
           <pixmap resource="mobii.qrc">:/misc/Resources/profile_default.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="firstnameLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="firstNameEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lastnameLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lastNameEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="societyLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>Société</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="societyEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="persoMobileLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Numéros:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="phoneList">
            <property name="styleSheet">
             <string notr="true"> padding: 1px 1px 1px 1px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="persoMailLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>E-mails:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="birthdayLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Date de naissance</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDateEdit" name="birthdayEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="notesLabel">
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QTextEdit" name="notesTextEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>130</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QListWidget" name="mailList"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updateContactButton">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Valider les changements</string>
       </property>
       <property name="icon">
        <iconset resource="mobii.qrc">
         <normaloff>:/buttons/Resources/okay.png</normaloff>:/buttons/Resources/okay.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteContactButton">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Supprimer ce contact</string>
       </property>
       <property name="icon">
        <iconset resource="mobii.qrc">
         <normaloff>:/buttons/Resources/remove.png</normaloff>:/buttons/Resources/remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mobii.qrc"/>
 </resources>
 <connections/>
</ui>
